src/main/java/org/kainos/ea/cli/LoginRequest.java

```diff
@@ -0,0 +1,44 @@
+package org.kainos.ea.cli;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class LoginRequest {
+
+  private String email;
+  private String password;
+  @JsonIgnore
+  private String passwordHash;
```

Comment:
Why is the `passwordHash` and ignored? It shouldn't be part of the `LoginRequest` from the frontend

Reply:
We are not wanting to capture this so I have used @JsonIgnore -> Works really well. However, we want to keep information in the same object for later processing. The thought was that this would limit the amount of objects created and what we do is simple use the setter to set this when required later in processing.

Reply:
The `passwordHash` should never leave the backend, or be passed to it by the frontend